backend/src/controllers/userController.js

```
@@ -375,4 +381,165 @@ export async function updateNotificationPreferences(req, res) {
+      action_required: 'Operations team has been notified and will complete the process.'
+    },
+    contact_support: true
+  });
+
+// Note: Removed the now-unreachable 200 success response at lines 521-525
+    
+    res.status(200).json({ 
+      success: true, 
+      message: 'Account and all associated data have been permanently deleted.',
+      audit_id: deletionResult?.audit_id
+    });
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)